src/app/public/modules/toolbar/toolbar-item.component.ts

```diff
@@ -2,6 +2,10 @@ import {
   Component
 } from '@angular/core';
 
+/**
+ * Specifies a container for an item in the toolbar.
+ * @required
```

Comment:
We don't need to use the `@required` annotation on component descriptions, only properties. While I admit it makes sense to use this from our docs output perspective, you can't "require" a component in the jsDoc sense. If we want to show this is required in the docs output, then we'd need to write it out in the description or figure out a better way to show this.

Reply:
We removed that sentence from the summary because it's an implementation detail, and the Design decision was to not include implementation details in the summary. I still think we're fine here and will resolve this comment thread, but let me know if you think we need to add something here, Alex. Thanks!